timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -93,7 +93,7 @@
     DynamicOptimizedTheta=75,
     AutoETS=70,
     AutoARIMA=60,
-    Chronos=55,
+    Chronos=70,
```

Comment:
Let's keep the old value so that the local models are all grouped together and Chronos is trained immediately afterwards?